abandoned-cart-emails.yml

```diff
@@ -0,0 +1,477 @@
+        notify_at_minutes:
+          $ref: '#/components/schemas/NotifyAtMinutes'
+        coupon_code:
+          $ref: '#/components/schemas/CouponCode'
+        template:
+          $ref: '#/components/schemas/Template'
+    AbandonedCartSettings:
+      title: AbandonedCartSettings
+      type: object
+      x-examples:
+        Inherit global channel items:
+          use_global: true
+      properties:
+        use_global:
+          type: boolean
+          description: Indicates inheritance state
+      required:
+        - use_global
+    AbandondedCartEmailPayload:
+      type: object
+      x-examples:
+        Update data:
+          coupon_code: FF11-22X4
+          notify_at_minutes: 60
+          template:
+            subject: 'Complete your purchase at {{ store.name }}'
+            body: 'Complete your purchase <a href="{{ notification.checkout_link }}">{{notification.checkout.link}}'
+            translations:
+              - locale: en
+              - keys:
+                  hello_phrase: Welcome back
+        Enable email:
+          is_active: true
+        Disable email:
+          is_active: false
+      properties:
+        is_active:
+          type: boolean
+        coupon_code:
+          $ref: '#/components/schemas/CouponCode'
+        notify_at_minutes:
+          $ref: '#/components/schemas/NotifyAtMinutes'
+        template:
+          $ref: '#/components/schemas/Template'
+      title: ''
+    Template:
+      title: Template
+      type: object
+      properties:
+        subject:
+          type: string
+          example: '''Complete your purchase at {{ store.name }}'''
+          maxLength: 255
+        body:
+          type: string
+          example: '''Complete your purchase <a href="{{ notification.checkout_link }}">{{notification.checkout.link}}'''
+        translation:
+          type: array
+          items:
+            $ref: '#/components/schemas/TranslationDefinition'
+      required:
+        - subject
+        - body
+        - translation
+    TranslationDefinition:
+      title: TranslationDefinition
+      type: object
+      x-examples:
+        example-1:
+          locale: en
+          keys: 
+            hello_phrase: Welcome back
+      properties:
+        locale:
+          type: string
+          example: en
+          description: 'Locale code for this language, such as ''en'', ''en-us'', ''fr-ca''.'
+        keys:
+          type: string
+          phrase_id: phrase_translation
+          description: |-
+            Language keys for the template. User-defined. Should match any lang keys used in the template.
+        additionalProperties:
+          nullable: true
+      required:
+        - locale
+        - keys
+    NotifyAtMinutes:
+      type: integer
+      title: NotifyAtMinutes
+      description: 'Describes when this email will be sent, in minutes. Only supports certain intervals described in the enum.'
+      example: 60
+      enum:
+        - 60
+        - ' 120'
```

Comment:
Hi @traci707 can you change all enums here from line 403 to 434?
They should be integers and not strings.